debian/rules

```diff
@@ -13,9 +13,9 @@ endif
 
 DEBVERS := $(shell dpkg-parsechangelog | sed -n -e 's/^Version: //p')
 VERSION := $(shell echo '$(DEBVERS)' | sed -e 's/^[\[:digit:]]*://' -e 's/[-].*//')
-MAJOR := $(shell echo '$(VERSION)' | awk -F . {'print $$1'})
-MINOR := $(shell echo '$(VERSION)' | awk -F . {'print $$2'})
-REV := $(shell echo '$(VERSION)' | awk -F . {'print $$3'})
+MAJOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$1'})
+MINOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$2'})
+REV := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$3'})
```

Comment:
This is because I switched to using a "+" for the version, eg `1.4.1+79~gfaea7dd`. This means 79 commits after the 1.4.1 tag and this seems to be what's recommended for git snapshots these days. This is found using `git --describe --tags`